### PR TITLE
Update initial-deployment.md

### PR DESCRIPTION
**Pull request by gecube**
No need to clone the repo. The helm chart itself is enough

### CODE REVIEW

**Comment by blinkov** [comment]
@gecube, why do you think this step needs to be removed?

**Comment by gecube** [comment]
If we look further:

https://ydb.tech/docs/en/devops/kubernetes/initial-deployment#deploy-storage-nodes

```bash
kubectl apply -f samples/storage-block-4-2.yaml
```

there is no need to pass a local file. It would be easy to pass a remote file from github repo. Please refer to https://stackoverflow.com/questions/53498449/passing-manifest-file-from-remote-url-with-kubectl-command
It is very interesting that kubectl can consume URL with YAML manifest directly.

**Comment by kobzonega** [comment]
@gecube

Yes, it seems that cloning the GitHub repository is unnecessary for the initial deployment with minimal requirements. However, as you can see below, we use the samples directory to apply Kubernetes manifests. So, if we remove this part, we also need to adjust the kubectl apply commands, as you suggested:

> There is no need to pass a local file. It would be easier to pass a remote file from the GitHub repository.

To implement these changes, please update the local paths to remote files to ensure users have a correct initial deployment flow with the appropriate commands

**Comment by gecube** [comment]
I rebased. And added an appropriate commands.
Also I noted that the name of file was incorrect here:

```
kubectl apply -f samples/storage-mirror-3-dc.yaml
```

the file is called `samples/storage-mirror-3dc.yaml`

I am kindly asking to check the PR and merge it.

**Comment by gecube** [comment]
Also I want to mention that the repo is really... large... and it takes the infinite time to fully pull it. A lot of stale branches etc. Is there any option to optimise it? I am totally sure that there is no need in 301 (!) branch in main repo!

File: ydb/docs/en/core/devops/kubernetes/initial-deployment.md [hunk 1]
```diff
@@ -143,15 +137,15 @@ Apply the manifest for creating storage nodes:
...
+  kubectl apply -f https://raw.githubusercontent.com/ydb-platform/ydb-kubernetes-operator/master/samples/storage-block-4-2.yaml
   ```
 
   This will create 8 {{ ydb-short-name }} storage nodes that persist data using erasure coding. This takes only 50% of additional storage space to provide fault-tolerance.
 
 - mirror-3-dc
 
   ```bash
-  kubectl apply -f samples/storage-mirror-3-dc.yaml
+  kubectl apply -f https://github.com/ydb-platform/ydb-kubernetes-operator/blob/master/samples/storage-mirror-3dc.yaml
```
Comment (blinkov) [inline ydb/docs/en/core/devops/kubernetes/initial-deployment.md:148]:
```suggestion
  kubectl apply -f https://raw.githubusercontent.com/ydb-platform/ydb-kubernetes-operator/master/samples/storage-mirror-3dc.yaml
```

**Comment by blinkov** [comment]
@gecube thank you for the contribution!

> Also I want to mention that the repo is really... large... and it takes the infinite time to fully pull it. A lot of stale branches etc. Is there any option to optimise it? I am totally sure that there is no need in 301 (!) branch in main repo!
> 
> ```shell
> gaal@bigsur ydb % du -s -h .
> 2.8G	.
> ```

The number of branches shouldn't affect the total size much as most of the changes are happening in `main`. YDB's codebase exists for over a decade and has a pretty high commit velocity. For example, we're currently at the 12th place at https://ossrank.com by commit velocity. There are no known long-hanging fruits for dramatically reducing this size, and any significant improvements will likely need rewriting the `git` history, which is tricky.

**Comment by gecube** [comment]
@blinkov thank you for your comments and suggestions. Regarding code base size... maybe it is possible to split docs and main code... no idea. Please try to open this question in your team internally.

**Comment by blinkov** [comment]
> Please try to open this question in your team internally.

I've already done that. The documentation is like 50mb and it is in the same repo because it is deployed to the website from the same branches that are used to build stable binaries. The largest folder by size is https://github.com/ydb-platform/ydb/tree/main/contrib, which holds third-party dependencies. Package managers for C++ aren't as common as for other programming languages, and many of the dependencies are custom forks anyway.

**Review by blinkov — APPROVED** [review]

**Pull request merged**